js/task_1.js

```diff
@@ -0,0 +1,27 @@
+//Лизе нужно написать функцию, которая возвр имя пользователя. Она влюблена в Mark, поэтому его приветствуем иначе
+
+const sayHello = function(name) {
+    if (name === "Mark") {
```

Comment:
What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue